src/components/browse/singleDatasetView/singleDatasetViewAccordion/singleDatasetViewAccordion.tsx

```diff
@@ -11,20 +11,20 @@ const SingleDatasetViewAccordion = (props: SingleDatasetViewAccordionProps) => {
 
   return (
     <Accordion>
-      <Accordion.Item className="mb-4" eventKey="0">
+      <Accordion.Item className="mb-4 border-0" eventKey="0">
         <Accordion.Button className="bg-secondary py-2 text-white rounded-0">
           Experiment Summary
         </Accordion.Button>
         <Accordion.Body className="pt-4">
-          <Table bordered hover className="fs-8" size="sm">
-            <thead>
+          <Table hover className="fs-8" size="sm">
+            <thead className="border-light-alternative border-1">
               <tr>
-                <th className="fs-7 w-25">Experiment</th>
-                <th className="w-25">Alias</th>
-                <th>Number of Samples</th>
+                <th className="w-25">Experiment ID</th>
+                <th className="w-25">Description</th>
+                <th>Experiment Type</th>
```

Comment:
it should be close to the right side or put a second column. For the case of the long description.
It should be changed for the sample summary as well